search.go

```diff
@@ -366,7 +365,8 @@ func (l *Conn) Search(searchRequest *SearchRequest) (*SearchResult, error) {
 	result := &SearchResult{
 		Entries:   make([]*Entry, 0),
 		Referrals: make([]string, 0),
-		Controls:  make([]Control, 0)}
+		Controls:  make([]Control, 0),
+	}
```

Comment:
I don't mind trivial whitespace changes like this, but I'd prefer if we keep them consistent between the root version and the `v3/` version